marshmallow/validate.py

```diff
@@ -214,10 +216,27 @@ class Length(Range):
     message_min = 'Shorter than minimum length {min}.'
     message_max = 'Longer than maximum length {max}.'
     message_all = 'Length must be between {min} and {max}.'
+    message_equal = 'Length must be {equal}.'
+
+    def __init__(self, min=None, max=None, error=None, equal=None):
```

Comment:
Great! I wish I had done this, but I didn't want to break the backward compatibility in a first moment.